project2/styles.css

```
@@ -0,0 +1,178 @@
+/* FONTS
+	- default body text  sofia-pro, sans-serif;
+    - date & cite        orpheuspro, serif;
+*/
+
+/* DISPLAY SETTINGS & UTILITIES */
+
+* {
+    box-sizing: border-box;
+}
+
+[data-visible="false"] {
+    display: none !important;
+}
+
+.u-button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+/* MAIN SECTIONS */
+
+body {
+    background-color: rgba(255, 241, 206, 1);
+    color: black;
+    font-family: sofia-pro, sans-serif;
+    font-weight: 400;
+}
+
+header {
+    background-color: rgba(166, 59, 50, 1);
```

You only need `rgba` if you're using that alpha; for this, you can do without:
```suggestion
    background-color: rgb(166, 59, 50);
```